ElasticFetch.js

```diff
@@ -0,0 +1,51 @@
+"use strict"
+
+class ElasticFetch {
+  constructor(baseUrl, indexName, transport) {
+    this.ressource = [baseUrl, indexName].join("/")
+    if (transport === "function") {
+      this._fetch = transport || fetch
+    }
+  }
+
+  /**
+   * Make HTTP requests to ressource
+   * @param {String} ressource the resource that you wish to fetch
+   * @param {Object} init custom settings that you want to apply to the request (method, headers ...)
+   * @returns Promise
+   */
+  fetch(init = {}, ressource = "_search") {
+    init.headers = {
+      ...init.headers,
+      ...{
+        "Content-Type": "application/json",
```

Comment:
Do we need this ?

```suggestion
        "Content-Type": "application/json;charset=utf-8'",
```

Reply:
I don't know if it's necessary or implicit, there is a small quote mistake on your suggestion @sbidoul 
```suggestion
        "Content-Type": "application/json; charset=UTF-8",
```